cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -134,15 +140,24 @@ public void execute() {
         }
 
         String[] args;
+        Boolean isModule = false;
+
         if (this.argList == null) {
             args = new String[0];
             this.projectPath = Paths.get(System.getProperty(ProjectConstants.USER_DIR));
         } else if (this.argList.get(0).startsWith(RuntimeConstants.BALLERINA_ARGS_INIT_PREFIX)) {
             args = argList.toArray(new String[0]);
             this.projectPath = Paths.get(System.getProperty(ProjectConstants.USER_DIR));
         } else {
-            args = argList.subList(1, argList.size()).toArray(new String[0]);
-            this.projectPath = Paths.get(argList.get(0));
+            if (ProjectUtils.isBallerinaProject(this.projectPath) &&
```

Comment:
should we also check whether argList size is only one, if more than 1 we can throw a warning.

Reply:
Added a check for arguments greater than one. Added a test case also